content/en/blog/_posts/2023-12-05-kubernetes-1.29.md

```diff
@@ -0,0 +1,166 @@
+
+This release consists of <xx> enhancements. Of those enhancements, <xx> are entering Alpha, <xx> have graduated to Beta, and <xxx> have graduated to Stable.
+
+## Release Theme And Logo
+    
+Kubernetes v1.29: xxxxxx
+
+The theme for Kubernetes v1.29 is xxxx.
+
+## What's New in [sig-storage](https://github.com/kubernetes/community/tree/master/sig-storage) 
+
+### Generally Available: ReadWriteOncePod PersistentVolume access mode
+
+In Kubernetes, volume [access modes](/docs/concepts/storage/persistent-volumes/#access-modes)
+are the way you can define how durable storage is consumed. These access modes are a part of the spec for PersistentVolumes (PVs) and PersistentVolumeClaims (PVCs).
+
+Before v1.22, Kubernetes offered three access modes for PVs and PVCs:
+	•	ReadWriteOnce – the volume can be mounted as read-write by a single node
+	•	ReadOnlyMany – the volume can be mounted read-only by many nodes
+	•	ReadWriteMany – the volume can be mounted as read-write by many nodes
+
+The ReadWriteOnce access mode restricts volume access to a single node, which means it is possible for multiple pods on the same node to read from and write to the same volume. This could potentially be a major problem for some applications, especially if they require at most one writer for data safety guarantees.
+    
+To address this problem, a fourth access mode ReadWriteOncePod was introduced as an Alpha feature in v1.22 for CSI volumes. If you create a pod with a PVC that uses the ReadWriteOncePod access mode, Kubernetes ensures that pod is the only pod across your whole cluster that can read that PVC or write to it. In v1.29, this feature became Generally Available.
+
+### Generally Available: Node Expand Secret for CSI Driver
+    
+In Kubernetes, a volume expansion operation may include the expansion of the volume on the node side which involves filesystem resize. Some CSI drivers require secrets (for example: a credential for accessing a SAN fabric) during the node expansion for the following use cases:
+	•	When a PersistentVolume represents encrypted block storage (for example using LUKS) you need to provide a passphrase in order to expand the device.
+	•	For various validations at time of node expansion, the CSI driver needs to have credentials in order to communicate with the backend storage system. 
+
+To meet this requirement, a CSI Node Expand Secret feature was introduced in Kubernetes v1.25 release. This allows an optional secret field to be sent as part of the NodeExpandVolumeRequest by the CSI drivers so that node side volume expansion operation can be performed with the underlying storage system. In Kubernetes 1.29 release, this feature became generally available.
```

Comment:
```suggestion
To meet this requirement, a CSI Node Expand Secret feature was introduced in Kubernetes 1.25. This allows an optional secret field to be sent as part of the NodeExpandVolumeRequest by the CSI drivers so that node volume expansion operation can be performed with the underlying storage system. In Kubernetes 1.29, this feature became generally available.
```